.github/workflows/ci.yaml

```
@@ -0,0 +1,85 @@
+name: MonoKickstart Ci
+on: [push]
+jobs:
+  build:
+    
+    name: Steam Runtime
+    runs-on: ubuntu-20.04
+    container: registry.gitlab.steamos.cloud/steamrt/sniper/sdk:beta
+    # TODO: Run on Rocky Linux, using buntu right now because I haven't checked the package names on RHEL yet
+    steps:
+      - uses: actions/checkout@v4
+        with:
+            submodules: true
+            path: monokickstart
+
+      - name: Build libgdiplus
+        run: |
+            sudo apt update
+            sudo apt install libgif-dev autoconf libtool automake build-essential gettext libglib2.0-dev libcairo2-dev libtiff-dev libexif-dev ninja-build mono-complete -y
+            git clone https://gitlab.winehq.org/mono/libgdiplus --depth=1 --recursive --shallow-submodules
+            cd libgdiplus
+            ./autogen.sh
+            make -j${nproc}
+
+      - name: Upload libgdiplus Artifact
+        uses: actions/upload-artifact@v4
+        with:
+            name: libgdiplus
+            path: libgdiplus/src/.libs/libgdiplus.so.0.0.0

```

Just to match what the CLR looks for:

```suggestion
            path: libgdiplus/src/.libs/libgdiplus.so.0

```

---

That's a symlink tough

---

I'll include both the file and the symlink, should be good enough I hope